### DISCUSSION TITLE
Latest

### DISCUSSION THREAD

**Discussion by Schwartzblat** [Schwartzblat/WhatsAppPatcher#72] [Announcements]
The ab test feature is now working fine and there are very cool results :)

<hr /><em>This discussion was created from the release <a href='https://github.com/Schwartzblat/WhatsAppPatcher/releases/tag/1.0.14'>Latest</a>.</em>

**Comment by Djrock79** [comment]
How does this patcher work? I downloaded the patched apk but there aren't any options or modded features. Is there a hidden menu from where I've to use the features?
